src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Storage.Interface.csproj

```diff
@@ -25,15 +25,15 @@
     <OutputPath>bin\Release\</OutputPath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-  </ItemGroup>
-
   <ItemGroup>
     <Content Include="Storage.Interface.xml">
       <Pack>true</Pack>
       <PackagePath>\</PackagePath>
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Also add an explicit framework reference inside the item group, since this project targets the "Microsoft.NET.Sdk" and not "Microsoft.NET.Sdk.Web" (which implicit reference the framework). 
<FrameworkReference Include="Microsoft.AspNetCore.App" />

[Ref. this Microsoft article on migration 2.2 => 3.0](https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.0&tabs=visual-studio#framework-reference)